src/lib/form/fieldManager.ts

```diff
@@ -62,6 +68,13 @@ export class FieldManager extends AbstractFormNode {
     );
   }
 
+  public serialise(): FieldJSON {
+    const value = this._value;
+    const errorMessages = this.errorMessages();
+
+    return { value, errorMessages };
+  }
+
```

Comment:
No change to `serialise()` or `shouldValidate()`, I just re-arranged the methods so public methods appear above private.